src/app/mail/mail-sidebar/compose-mail/compose-mail.component.ts

```diff
@@ -895,4 +892,17 @@ export class ComposeMailComponent implements OnInit, AfterViewInit, OnDestroy {
       this.closeOSK();
     }
   }
+
+  onFilesDrop(event) {
+    if (!this.draftMail || !this.draftMail.id) {
+      this.updateEmail();
+    }
+    const files = event.dataTransfer.files;
```

Comment:
Call `this.onImagesSelected(files)` here instead of duplicating code.

Reply:
@Waseem826 this.onImagesSelected attach files inline.
Please check two functions: onImagesSelected & onFilesSelected.